### PR TITLE
Run most tests with `-sCLOSURE_WARNINGS=error`

### PR DESCRIPTION
**Pull request by sbc100**
This is (hopefully) the last the sequence of change to route out all the
closure warnings in our codebase.

See #12728

### CODE REVIEW

File: src/library_browser.js [hunk 1]
```diff
@@ -339,12 +339,10 @@ var LibraryBrowser = {
           alpha: false,
 #if MIN_WEBGL_VERSION >= 2
           majorVersion: 2,
-#else
-#if MAX_WEBGL_VERSION >= 2 // library_browser.js defaults: use the WebGL version chosen at compile time (unless overridden below)
+#elif MAX_WEBGL_VERSION >= 2 // library_browser.js defaults: use the WebGL version chosen at compile time (unless overridden below)
```
Comment (sbc100) [inline src/library_browser.js:342]: This is unrelated .. but probably not worth splitting out.

File: tests/common.py [hunk 2]
```diff
@@ -827,7 +833,10 @@ def get_library(self, name, generated_libs, configure=['sh', './configure'],
     build_dir = self.get_build_dir()
     output_dir = self.get_dir()
 
-    emcc_args = self.get_emcc_args()
+    # get_library() use to compiler libraries, and not link executables,
```
Comment (kripken) [inline tests/common.py:836]:
```suggestion
    # get_library() is used to compile libraries, and not link executables,
```

**Review by kripken — APPROVED** [review]
